content/docs/command-reference/pull.md

```diff
@@ -107,89 +107,146 @@ reflinks or hardlinks to put it in the workspace without copying. See
+[Get Started](/doc/tutorials/get-started). Then we can see what happens with
+`dvc pull`.
+
+<details>
+
+### Click and expand to setup the project
+
+Start by cloning our example repo if you don't already have it:
 
 ```dvc
-$ dvc remote add -d r1 ssh://_username_@_host_/path/to/dvc/remote/storage
-$ dvc remote list
-r1	ssh://_username_@_host_/path/to/dvc/remote/storage
+$ git clone https://github.com/iterative/example-get-started
+$ cd example-get-started
 ```
 
-> DVC supports several
-> [remote types](/doc/command-reference/remote/add#supported-storage-types).
+</details>
 
-Having some images and other files in remote storage, we can pull all changed
-files from the current Git branch:
+The workspace looks almost like in this
+[pipeline setup](/doc/tutorials/pipelines):
 
 ```dvc
-$ dvc pull --remote r1
+.
+├── data
+│   └── data.xml.dvc
+├── evaluate.dvc
+├── featurize.dvc
+├── prepare.dvc
+├── train.dvc
+└── src
+│   └── ...
+└── train.dvc
 ```
 
-We can download specific files that are <abbr>outputs</abbr> of a specific
-DVC-file:
+We can now just run `dvc pull` to download the most recent `model.pkl`,
+`data.xml`, and other DVC-tracked files into our local <abbr>cache</abbr> and
+<abbr>workspace</abbr>:
 
 ```dvc
-$ dvc pull data.zip.dvc
+$ dvc pull
+
+$ tree example-get-started/
+example-get-started/
+├── data
+│   ├── data.xml
+│   ├── data.xml.dvc
+│   ├── features
+│   │   ├── test.pkl
+│   │   └── train.pkl
+│   └── prepared
+│       ├── test.tsv
+│       └── train.tsv
+├── ...
 ```
 
-In this case we left off the `--remote` option, so it will have pulled from the
-default remote. The only files considered in this case are what is listed in the
-`out` field of the DVC-file `targets`.
+We can download specific <abbr>outputs</abbr> of a single DVC-file:
+
+```dvc
+$ dvc pull prepare.dvc
+```
 
 ## Example: With dependencies
 
-Demonstrating the `--with-deps` option requires a larger example. First, assume
-a [pipeline](/doc/command-reference/pipeline) has been setup with these
+Our [pipeline](/doc/command-reference/pipeline) has been setup with these
 [stages](/doc/command-reference/run):
 
 ```dvc
-$ dvc pipeline show
-
-data/Posts.xml.zip.dvc
-Posts.xml.dvc
-Posts.tsv.dvc
-Posts-test.tsv.dvc
-matrix-train.p.dvc
-model.p.dvc
-Dvcfile
+$ dvc pipeline show evaluate.dvc
+data/data.xml.dvc
+prepare.dvc
+featurize.dvc
+train.dvc
+evaluate.dvc
 ```
 
 Imagine the remote storage has been modified such that the data in some of these
 stages should be updated in the <abbr>workspace</abbr>.
 
 ```dvc
-$ dvc status --cloud
-
-  deleted:            data/model.p
-  deleted:            data/matrix-test.p
-  deleted:            data/matrix-train.p
+$ dvc status -c
+    deleted:            data/features/test.pkl
+    deleted:            data/features/train.pkl
+    deleted:            model.pkl
+    deleted:            data/prepared/test.tsv
+    deleted:            data/prepared/train.tsv
+    deleted:            data/data.xml
```

Comment:
Let's just use a few missing data instead deleting these lines:
```suggestion
```

Reply:
Wait... This isn't right. `dvc status -c` checks what's on the remote. That would be relevant for `dvc push`, not `dvc pull`. Maybe this is an error in our existing docs! Please actually reproduce the example with a local remote to make sure things make sense.

Reply:
It isn't required for `pull` but here is it used just to show which files would be downloaded.
This is mentioned in pull [description](https://dvc.org/doc/command-reference/pull#description):
>The command `dvc status -c` can list files referenced in current DVC-files, but missing in the cache. It can be used to see what files dvc pull would download.

Reply:
You're right. I had a lapsus there.